cmd/mungedocs/unversioned_warning.go

```diff
@@ -71,11 +76,15 @@ func updateUnversionedWarning(file string, mlines mungeLines) (mungeLines, error
 		// No warnings on release branches
 		return mlines, nil
 	}
+	linkToPreviousDoc := true
+	if hasLine(mlines, "<!-- TAG NEW -->") {
```

Comment:
I'm not sure. It seems the BEGIN MUNGE and END MUNGE are added by the munger to enclose the contents added by the munger. In our case, the TAG NEW is manually added, so it probably doesn't need the BEGIN or END.